dracut/anaconda-modprobe.sh

```diff
@@ -30,6 +32,5 @@ MODULE_LIST+=" raid0 raid1 raid5 raid6 raid456 raid10 linear dm-mod dm-zero  \
               sha256 lrw xts "
 
 for m in $MODULE_LIST; do
-    modprobe $m &>/dev/null
+    modprobe $m && debug_msg "$m was loaded" || debug_msg "$m was NOT loaded"
```

Comment:
Could this be an `if-else` instead of this please.

Reply:
Fixed.